### DISCUSSION TITLE
difference in JSON data and Dashboard display

### DISCUSSION THREAD

**Discussion by Tobi7241** [beep-projects/SystaPi#21] [General]
there is an difference in JSON data and Dashboard display:

![grafik](https://github.com/beep-projects/SystaPi/assets/62337620/b1b70dc3-5902-4993-a2b8-8441a5620a69)

![grafik](https://github.com/beep-projects/SystaPi/assets/62337620/08c12057-4282-49d8-971b-c29a0bf8ff6c)

![grafik](https://github.com/beep-projects/SystaPi/assets/62337620/a3457407-41ef-4f41-b4f2-8e9d1e15fd08)

from the existing Paradigma status email records and S-Touch reading, the JSON number 29337 should be right.
Can someone confirm ? the divider "x/10.0" seems wrong .

Tobias

**Comment by beep-projects** [comment]
Hi, I think the divider is wrong.  At least in the paradigma remote portal code both have a factor of 1 (and not 0.1).
